src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -6084,11 +6082,79 @@ void CEndlessUsbToolDlg::DelayDeleteFolder(const CString &folder)
 	}
 }
 
+static bool _lowerCaseExeNameFindSubstring(CStringW exePath, const wchar_t* substring, bool* isSuffix)
```

Comment:
This is declared to return `bool` but in fact returns an `int` and its two callers assume its return value is an `int`. MSVC emits the following warnings:

```
1>c:\users\wjt\source\repos\rufus\src\endless\endlessusbtooldlg.cpp(6094): warning C4800: 'int': forcing value to bool 'true' or 'false' (performance warning)
1>c:\users\wjt\source\repos\rufus\src\endless\endlessusbtooldlg.cpp(6113): warning C4804: '>': unsafe use of type 'bool' in operation
```